d3a_api_client/utils.py

```diff
@@ -276,14 +278,17 @@ def log_bid_offer_confirmation(message):
 
 def get_simulation_info(simulation_id=simulation_id_from_env, domain_name=domain_name_from_env,
                         websockets_domain_name=websocket_domain_name_from_env):
-    if os.environ['JSON_FILE_PATH'] != "":
+    if os.environ.get('JSON_FILE_PATH', "") != "":
         with open(os.environ['JSON_FILE_PATH']) as json_file:
             simulation_info = json.load(json_file)
-            simulation_id = simulation_info['uuid']
-            domain_name = simulation_info['domain_name']
-            websockets_domain_name = simulation_info['web_socket_domain_name']
+        if set(simulation_info.keys()) != {"username", "name", "uuid", "domain_name",
+                                           "web_socket_domain_name", "global_settings", "registry"}:
```

Comment:
Probably premature, but we will need a validation method that checks for these keys.
This validator should live in the d3a-interface and should be called also before downloading the simulation-config-file. 
But please feel free to skip it. Maybe for some time when we finally settled on the in formations we put in the simulation-config-file.

Reply:
I just found my old comment 
https://github.com/gridsingularity/d3a-api-client/pull/94#discussion_r596171500
That you didnt reply to. What do you think?

Reply:
Sorry, after reading your comment again, realised that didn't fully adapted ie I did added an assertion logic to compare  keys if the read json is the right format but didn't abstracted it into a validator method, thanks for pointing it again, will abstract into a validation method and make part of `d3a-interface`